### ISSUE TITLE
Question: What is a final value?

### ISSUE THREAD

**Issue by hanneslueke** [Steve-Mcl/node-red-contrib-buffer-parser#22]
In your documentation you refer to something called final value. Could you elaborate what a final value semtically is, please?

**Comment by Steve-Mcl** [comment]
Can you clarify please (save me looking, loading source, scanning the readme, scanning the built-in docs etc)

a screenshot would do.

**Comment by hanneslueke** [comment]
![image](https://github.com/Steve-Mcl/node-red-contrib-buffer-parser/assets/152261405/58a9b04d-98ba-42fc-a5a9-83b9e4701af8)

**Comment by Steve-Mcl** [comment]
Ok, so I contextually, this information (and the options they describe) come after the other options in a row. So it refers to the fact that once the value has been extracted from the buffer, converted to the specified type, the quantity requested etc, these options are applied to those values. 

Tldr;
The final value you would be sent can be affected by the settings in the scale and mask operations. Example: if you have no mask and no scaling, you get the final value for that row. Example: if you have a scale of 0.5, the vinal value is halved.
